proto/evmos/incentives/v1/genesis.proto

```diff
@@ -0,0 +1,27 @@
+syntax = "proto3";
+package evmos.incentives.v1;
+import "evmos/incentives/v1/incentives.proto";
+
+import "gogoproto/gogo.proto";
+import "google/protobuf/duration.proto";
+
+option go_package = "github.com/tharsis/evmos/x/incentives/types";
+
+// GenesisState defines the module's genesis state.
+message GenesisState {
+    // module parameters
+    Params params = 1 [(gogoproto.nullable) = false];
+    // registered token pairs
+    repeated Incentive incentives = 2 [(gogoproto.nullable) = false];
+}
+
+// Params defines the erc20 module params
+message Params {
```

Comment:
missing: max incentive percent (should be 5%) that prevents incentives with allocation greater than the max to be created